docs/usage.md

```diff
@@ -585,11 +585,13 @@ This list is by no means exhaustive and it will depend on the specific analysis
 | [mpileup](https://www.htslib.org/doc/samtools-mpileup.html)                                             |  x  |  x  |   x    |    x    |   x   |    -    |
 | [Strelka](https://github.com/Illumina/strelka)                                                          |  x  |  x  |   x    |    x    |   -   |    x    |
 | [Manta](https://github.com/Illumina/manta)                                                              |  x  |  x  |   x    |    x    |   x   |    x    |
+| [indexcov](https://github.com/brentp/goleft/tree/master/indexcov)                                       |  x  |  -  |   x    |    x    |   x   |    x    |
```

Comment:
```suggestion
| [indexcov](https://github.com/brentp/goleft/tree/master/indexcov)                                       |  x  |  -  |   x    |    x    |   -   |    x    |
```

Reply:
@lindenb From the code it looks like tumor-only is currently not covered. Would remove it here for now

Reply:
Going to push this. Need to beat the tools release. Ain't nobody got time for a template update :D